docs/sql-reference/14-sql-commands/00-ddl/stream/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Stream
+---
+import IndexOverviewList from '@site/src/components/IndexOverviewList';
+import EEFeature from '@site/src/components/EEFeature';
+
+<EEFeature featureName='STREAM'/>
+
+### What is Stream?
+
+A stream in Databend is a dynamic and real-time representation of changes to a table. Streams are created to capture and track modifications to the associated table, allowing continuous consumption and analysis of data changes as they occur. 
+
+This is a quick example illustrating what a stream looks like. In the given scenario, a stream named 'my_first_st' is created for the 'test_stream' table and captures a data insertion:
+ 
+```sql title='Example:'
+CREATE TABLE test_stream(a INT);
+INSERT INTO test_stream VALUES(1);
+ALTER TABLE test_stream SET OPTIONS(change_tracking = TRUE);
+
+CREATE STREAM my_first_st ON TABLE test_stream;
+INSERT INTO test_stream VALUES(2);
+SELECT * FROM my_first_st;
+
+┌─────────────────┐
+│        a        │
+├─────────────────┤
+│               2 │
+└─────────────────┘
+```
+
+A Databend stream currently supports **Append-only** mode, allowing it to capture **both data insertion and deletion events**, as demonstrated in the following example:
+
+```sql title='Example continued:'
+INSERT INTO test_stream VALUES(3);
+SELECT * FROM my_first_st;
+
+┌─────────────────┐
+│        a        │
+├─────────────────┤
+│               2 │
+│               3 │
+└─────────────────┘
+
+DELETE FROM test_stream WHERE a = 2;
+SELECT * FROM my_first_st;
+
+┌─────────────────┐
+│        a        │
+├─────────────────┤
+│               3 │
+└─────────────────┘
+```
+
```

Comment:
I think we no need to introduce this part, since create stream on table will automatically enable it.
cc @zhyass

Reply:
CREATE TABLE test (
    book_id INT,
    title VARCHAR(255),
    author VARCHAR(255),
    publication_year INT
);

CREATE STREAM st ON TABLE test;

SQL Error [2733]: Query failed (#25571069-bed7-4fb1-8012-b5219aa37027): Change tracking is not enabled for table 'default.test'

Reply:
> I think we no need to introduce this part, since create stream on table will automatically enable it. cc @zhyass

It's not automatically enabled yet, thinking about making it a bit smarter.